docs/DeveloperGuide.md

```diff
@@ -903,23 +915,23 @@ This section contains the user stories for the different versions of our product
      
 2. Un-marking an item
 
-   *Assumption: there are 5 or more items in the list. The 5th item has a description of 'lemons', price of '$4.00' and quantity of '6'*
+   *Assumption: there are more than 5 and less than a hundred items in the list.*
        
    i. Test case: <code>UNMARK 5</code> 
 
-         Expected: The fifth item in the list is unmarked as bought, denoted as [0].
+         Expected: The fifth item in the list is un-marked as bought, denoted as [X].
```

Comment:
Maybe should be "marked as not bought yet". "unmarked as bought" might sound a bit funny

Reply:
Okiesz! Changed, thanks!!!